Testing/Functional/Products/FunctionalTestUtils.ps1

```diff
@@ -86,6 +86,22 @@ function LoadProviderExport() {
   $ProviderExport
 }
 
+function Write-ScubaConfig() {
+  <#
+    .SYNOPSIS
+      Private helper function to write scuba config to a file.
+  #>
+  param(
+      [Parameter(Mandatory = $true)]
+      [ValidateNotNullOrEmpty()]
+      [hashtable]
+      $Config
+  )
+
+  $ScubaTestConfigPath = Join-Path -Path "C:\Temp" -ChildPath "ScubaTestConfig.json"
```

Comment:
@crutchfield will address this in a separate PR. I borrowed his code for this PR since it contained some fixes that were still not committed to Emerald.